test.sh

```diff
@@ -10,9 +10,6 @@ test_err() {
 }
 
 test_js() {
-    if [[ ! -x ./node_modules/karma/bin/karma || ! -x ./node_modules/jshint/bin/jshint ]]; then
-        npm install
-    fi
     npm run lint -- "${jsfiles[@]}"
     if [[ ! ${sirepo_test_no_karma:-} ]]; then
         npm run test
```

Comment:
This needs to be:

```sh
karma start etc/karma-conf.js
```

Reply:
0d0cbd9727a002b1ea964084a02b7d51b8ab1f25